src/lib/secrets.ts

```diff
@@ -32,6 +32,11 @@ export enum Secret {
      * We only need one token in existence, so delete old tokens at: https://www.npmjs.com/settings/tokens
      */
     NPM_TOKEN,
+    /**
+     * Token used to publish packages to Github.
+     * This *could* be the same as GITHUB_ACCESS_TOKEN, but I think it's better if they remain separate.
```

Comment:
I'm not actually sure about this. It might be OK to have one token which can

1. publish
2. search for issues
3. comment on issues

I'll have to think about security a bit before deciding. I think it's fine to start with them separate.